github/ci/jenkins-master/templates/04-jobs.groovy

```diff
@@ -110,6 +110,21 @@ cd go/src/kubevirt.io/kubevirt && bash automation/test.sh 2>&1 | tee ${WORKSPACE
                 }
             }
         }
+        postBuildScripts {
+            steps {
+                shell('''#!/bin/bash
+set +e
+
+if [ -n "$(docker ps -q -f status=exited)" ]; then
+	docker rm $(docker ps -q -f status=exited)
+fi
+
+if [ -n "$(docker volume ls -qf dangling=true)" ]; then
```

Comment:
We are deliberately keeping some volumes, although they will not be used. I am not sure if this would not also remove these volumes.